modules/yarn.lock

```diff
@@ -2373,10 +2365,10 @@
     "@types/react" "*"
     hoist-non-react-statics "^3.3.0"
 
-"@types/html-minifier-terser@5.1.0", "@types/html-minifier-terser@^6.0.0":
```

Comment:
Those changes on yarn.lock seems weird. Could you run an `npx yarn-deduplicate` inside `modules/` folder?

Reply:
You did 

> Running yarn add alloyeditor@2.14.6 generated yarn.lock changes.

What I use to do is:

1. Go to [alloyeditor-web module](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-alloyeditor-web/package.json#L3) and change the version here
2. Go to `modules/` and run `yarn install` and then `npx yarn-deduplicate`
3. Go back to `alloyeditor-web` module and `gw clean deploy` there to see if everything is on the right place testing the UI :)

Reply:
Force-pushed changes!

I ran `yarn install` and `npx yarn-deduplicate`, it did rollback several `yarn.lock` changes. I added this to docs, to the new [Updating Alloy Editor in DXP section](https://github.com/liferay/alloy-editor/wiki/Contributing#updating-alloy-editor-in-dxp).

As for UI, the new code is correctly applied, we can make the release.

But! @orsolyaDekany during my testing I noticed there is a potential problem with the `ref` solution. Attribute does get added, but it disappears after save. See gif:

![screenshot](https://user-images.githubusercontent.com/5435511/168055671-b100b189-0c73-4126-a318-b0f774425dd8.gif)

Reply:
Hi @markocikos , I was not able to reproduce it on my side, `rel` attribute is still there after publishing the blog or web content.
![2022-05-13-12-47-03](https://user-images.githubusercontent.com/69793575/168268320-bb924e68-85c2-4512-8039-8f409e9ef993.gif)
I tested the fix locally linking the AlloyEditor repo to Liferay Portal repo.
Is there something in Liferay Portal that removes the attribute? I am looking into it.

Reply:
I don't know, I cannot not reproduce it :slightly_smiling_face: . Maybe it's something weird on my side. Can you ask somebody else to try?

I tried on Chrome 101 and Firefox 100, Ubuntu 20

Reply:
Okay, we figure out that the issue is not reproducible on 7.3.x and 7.2.x, but it is on Master. Currently, I do not know why is the difference.